how_to_add_new_hotspot.md

```diff
@@ -0,0 +1,142 @@
+# How To Add A New Hotspot Into Nebra Firmware
+
+The Nebra firmware stack has been designed to support various different architectures and vendors. Hence it is very flexible by design. But this also gives the developers a responsibility about supplying correct hardware configuration.
+
+This configuration information has to be supplied in different places of the stack. Let's review them one by one.
+
+## Checklist for adding a new hotspot type
```

Comment:
Can you link to specific places in code that need to be changed?

Reply:
Done.

Reply:
Disregard some of the links I added. I see you added similar links in the subsections below. In that case, maybe each item in the checklist should link to the corresponding subsection below?

Reply:
That's a simple check list. I think the developer could easily find its way to the corresponding sections. Truce? :smile: